src/auro-datepicker.js

```diff
@@ -881,13 +826,11 @@ export class AuroDatePicker extends LitElement {
           part="dropdown">
           <div slot="trigger" class="dpTriggerContent" part="trigger">
             <${this.inputTag}
-              auro-input
               id="${this.generateRandomString(12)}"
               bordered
               class="dateFrom"
               ?required="${this.required}"
               noValidate
-              .error="${this.error}"
```

Comment:
Why aren't we passing down the error attribute any more? I assume this is being fully handled by validation now?

Reply:
Correct